static/css/docs.scss

```diff
@@ -11,20 +11,52 @@
 @include docs-wrapper;
 @include docs-toc;
 
-.documentation__nav h4 {
-  margin-top: 0.2em;
-  margin-bottom: 0.2em;
+.documentation__wrapper {
+  flex-direction: column;
+
+  @media only screen and (min-width : $breakpoint-medium) {
+    flex-direction: column;
+  }
 }
 
-.documentation__nav ul li:after {
-  height: 2.25em;
+.documentation__jump {
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-left: 20px;
+  padding-right: 20px;
+
+  @media only screen and (min-width : $breakpoint-medium) {
+    display: none;
+  }
+}
+
+.documentation__nav {
+  order: 1;
+
+  @media only screen and (min-width : $breakpoint-medium) {
+    order: 0;
+  }
```

Comment:
I'm in a bit of a rush. They want this to be demoed in half and hour and released tomorrow, and it's not like I'm adding this stuff to either `docs-vanilla-theme` or `vanilla-framework`.